src/vstart.sh

```diff
@@ -977,7 +991,8 @@ do_rgw()
     i=0
     for rgw in j k l m n o p q r s t u v; do
 	ceph_adm auth get-or-create client.rgw.$rgw mon 'allow rw' osd 'allow rwx' mgr 'allow rw' -o $CEPH_DEV_DIR/rgw.$rgw.keyring
-	run 'rgw' $RGWSUDO $CEPH_BIN/radosgw -c $conf_fn --log-file=${CEPH_OUT_DIR}/rgw.$rgw.log ${RGWDEBUG} --debug-ms=1 -n client.rgw.$rgw -k $CEPH_DEV_DIR/rgw.$rgw.keyring
+	echo start rgw on http://localhost:$((CEPH_RGW_PORT + i))
+	run 'rgw' $RGWSUDO $CEPH_BIN/radosgw -c $conf_fn --log-file=${CEPH_OUT_DIR}/rgw.$rgw.log ${RGWDEBUG} --debug-ms=1 -n client.rgw "--rgw_frontends=${rgw_frontend} port=$((CEPH_RGW_PORT + i))"
```

Comment:
this changes the instance name from `client.rgw.$rgw` to `client.rgw`, but we're still creating keys for both. we should probably remove the `ceph_adm auth get-or-create client.rgw.$rgw` line above. also, the keyring created for `client.rgw` uses `allow *` for everything (even mds), whereas these `client.rgw.$rgw` keys are more restrictive